src/style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 11px;
+  line-height: 13px;
+  display: flex;
+}
+
+p {
+  margin: 0;
+}
+
+.item {
+  flex: 0 0 200px ;
+  padding: 30px 14px 15px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.item__photo {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 45px;
+}
+
+.item__img {
+  width: 160px;
+  height: 134px;
+}
+
+.item__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-top: 0;
+  margin-bottom: 6px;
+}
+
+.item__cod {
+  margin-bottom: 15px;
+}
+
+.item__rate,
+.item__cost {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  align-items: center;
+}
+
+.item__stars {
+  display: flex;
+}
+
+.item__star {
+  background-image: url(images/star.svg);
+  height: 15px;
+  width: 15px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.item__stars .item__star:nth-child(-n + 4) {
```

Comment:
add rules for -n+1, -n+2 etc

Reply:
add modifier to item-stars, `item__stars--4` when you need 4 active stars and so on